src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -742,6 +742,7 @@ RestoreBackupSetUserName = User Name
 RestoreBackupSetExpiration = Expiration
 TheLastBackupTaken = The last backup taken ({0})
 NoBackupsetsToRestore = No backupset selected to be restored
+UnsupportedDeviceType = Unsupported device type {0} for engine edition {1}.
```

Comment:
This should take in parameters for these values

`UnsupportedDeviceType(string deviceType, string engineEdition) = Unsupported device type {0} for engine edition {1}.`

(I realize other strings in this file aren't doing it correctly, just ignore those for now and we can fix them later)